src/ngResource/resource.js

```diff
@@ -443,10 +443,14 @@ angular.module('ngResource', ['ng']).
           });
 
           params = params || {};
-          forEach(self.urlParams, function(_, urlParam) {
+          forEach(self.urlParams, function(paramInfo, urlParam) {
             val = params.hasOwnProperty(urlParam) ? params[urlParam] : self.defaults[urlParam];
             if (angular.isDefined(val) && val !== null) {
-              encodedVal = encodeUriSegment(val);
+              if (paramInfo.isQueryParamiValue === true) {
```

Comment:
NP :)

BTW, when you have submitted a PR, you don't need to push upstream or anything; pushing to your origin/<branch> will auto-update the PR (GitHub feature), so that's probably what happened.

Just ping me when you're done, so I take a final look. Thx for all the work so far !